tests/examples/test_big_models_with_accelerate.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+
+import pytest
+
+from tests.examples.utils import (
+    ReadMe,
+    copy_and_run_script,
+    gen_cmd_fail_message,
+    requires_gpu,
+    requires_gpu_count,
+    requires_gpu_mem,
+)
+
+
+@pytest.fixture
+def example_dir() -> str:
+    return "examples/big_models_with_accelerate"
+
+
+@pytest.mark.example
+class TestBigModelsWithAccelerate:
+    """
+    Tests for examples in the "big_models_with_accelerate" example folder.
+    """
+
+    def test_readme_has_install_command(self, example_dir: str):
+        """
+        Test that the README has a valid install command.
+        """
+        readme_path = Path.cwd() / example_dir / "README.md"
+        readme = ReadMe(readme_path)
+
+        code = readme.get_code_block_content(position=1, lang="shell")
+        assert "pip install" in code
+
+        assert code.startswith("pip install llmcompressor")
+
+    @pytest.mark.parametrize(
+        ("script_filename", "visible_gpus"),
+        [
+            pytest.param("cpu_offloading_fp8.py", "0", id="cpu_offloading"),
+            pytest.param(
+                "multi_gpu_int8.py",
+                "",
+                id="multi_gpu_int8",
+                marks=[
+                    requires_gpu_mem(630),
+                    requires_gpu_count(2),
+                    pytest.mark.multi_gpu,
+                ],
+            ),
+            pytest.param(
+                "multi_gpu_int8_sequential_update.py",
+                "",
+                id="multi_gpu_int8_sequential_update",
+                marks=[requires_gpu_count(2), pytest.mark.multi_gpu],
+            ),
+        ],
+    )
+    @requires_gpu
```

Comment:
is the expectation that these will always require multiple GPUs? could we have separate annotations for number of gpus required?

Reply:
There are multiple marks and decorators being used with great specificity:

**skip decorators** (used to determine if a test should be skipped)
* `requires_gpu` — the test requires at least one GPU, skipped if not met
* `requires_gpu_count(NUM)` — the test requires at least NUM GPUs, skipped if not met
* `requires_gpu_mem(AMOUNT)` — the test requires at least AMOUNT GPU memory (in GB), skipped if not met

**markers** (used to control test selection)
* `example` — indicates the test is for an example (applied to all in this PR)
* `multi_gpu` — indicates the test requires multiple GPUs (this is used to select tests for the `k8s-h100-duo` run as none require more than two/exclude them from the `-solo` run)